charts/fybrik/values.yaml

```diff
@@ -477,20 +523,15 @@ opaConnector:
       # Relavent if tls is used.
       # certSecretName: "test-tls-opa-connector-certs"
       certSecretName: ""
-      # Name of kubernetes tls secret namespace that holds opa-connector certificate.
-      # certSecretNamespace: "fybrik-system"
-      certSecretNamespace: ""
-      # Name of kubernetes secret that holds the certificate authority (CA) certificate which is used
-      # by opa-connector to validate the connection to the manager if mtls is enabled.
-      # The provided certificates are used along with the certificates in the system CA certificate store.
+      # Name of kubernetes secret that holds the certificate authority (CA) certificates 
+      # which are used by opa-connector to validate the connection to the manager if
```

Comment:
to validate connections *from* the manager

do we need the secret name ?

Reply:
in the tls handshake, the manager sends its certificate to the connector. the name of the manager certificate is provided to the manager only.